ASUWebPlatforms/webspark-ci
# [2.14.0] WS2-2089: New Component: Animated Quote

### Description
New Component: Animated Quote

#### Solution 
Added new plugins with blockquote animated option for ckeditor
![ckeditor1](https://github.com/user-attachments/assets/07267b46-1f91-49d4-88fa-1e12fd17259e)
![ckeditor2](https://github.com/user-attachments/assets/bb5363e1-d43e-401b-82ca-0f9bdad99f99)
![ckeditor3](https://github.com/user-attachments/assets/de078e70-8f44-4da1-8171-b2e91f95adfc)


### Links

- [JIRA ticket](https://asudev.jira.com/browse/WS2-2089)
- 
Example PAGE : 
https://pr-570-webspark-ci.ws.asu.edu/blockquote-animated
### QA Steps

- Pull the branch locally into your existing env (do this instead of using the PR multidev so you can test the update hooks).
- Run `drush updb`. Update hook `webspark_update_10012` should run without error.
- Create or edit a Basic Page.
- Verify that the "Blockquote Animate" and "Highlight" options now appear on the CKEditor toolbar.
- Ensure the "Highlight" option is disabled by default.
- Add a new "Blockquote Animate."
- When you click on the blockquote content, confirm that the "Highlight" option becomes enabled. Then, add content as usual and verify the animated effect over the this new plugin content.
- In your local env, create a backup of your database as needed.
- Wipe the database, and perform a new installation.
- Create a new Basic page.
- Ensure the new plugin appear by default on basic html and full html option for Ckeditor.

### Checklist


### Verified in browsers 


### Screenshots



Note: Sections that are not applicable for this issue can be removed.
